console/web3ext/web3ext.go

```diff
@@ -1130,6 +1140,14 @@ web3._extend({
 			name: 'latestAnchoredBlockNumber',
 			getter: 'subbridge_getLatestAnchoredBlockNumber'
 		}),
+		new web3._extend.Property({
+			name: 'parentOperatorFeePayer',
```

Comment:
How about adding "get"?  
"parentOperatorFeePayer" -> "getParentOperatorFeePayer"

Reply:
We don't usually add `get` for properties. :)